catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/RemoveAllCommand.java

```diff
@@ -236,14 +247,16 @@ private boolean isAccidentalRemoval() {
       // use a message specific to cache and expired options
       final String warning =
           String.format(
-              "WARNING: This will permanently remove all %1$srecords from the %2$s. Do you want to proceed? (yes/no): ",
+              "WARNING: This will permanently remove all %srecords from the %s. Do you want to proceed? (yes/no): ",
```

Comment:
Updated output:
```
admin@root()> catalog:removeall --expired --cache
WARNING: This will permanently remove all expired records from the cache. Do you want to proceed? (yes/no): yes

Cache cleared in 0.006 seconds
admin@root()> log:tail
09:03:24,692 | INFO  | -expired --cache | ce.ddf.commands.catalog.RemoveAllCommand  131 | atalog-core-commands | Cache cleared in 0.006 seconds
09:03:24,693 | INFO  | -expired --cache | ce.ddf.commands.catalog.RemoveAllCommand  132 | atalog-core-commands | Cache cleared using the "catalog:removeall --cache" command
^C
admin@root()> log:clear
admin@root()> catalog:removeall --expired
WARNING: This will permanently remove all expired records from the Catalog. Do you want to proceed? (yes/no): yes

0 file(s) removed in 0.016 seconds
admin@root()> log:tail
09:03:49,872 | INFO  | oveall --expired | ce.ddf.commands.catalog.RemoveAllCommand  212 | atalog-core-commands | 0 file(s) removed in 0.016 seconds
09:03:49,872 | INFO  | oveall --expired | ce.ddf.commands.catalog.RemoveAllCommand  213 | atalog-core-commands | 0 files removed using the "catalog:removeall" command
```